stylesheets/base/_typography.scss

```diff
@@ -1,7 +1,112 @@
 /**
  * Basic typography style for copy text
  */
+
+// TODO: Convert to relative units using base font size instead of pixels
+
+//Import Google fonts
+@import url('https://fonts.googleapis.com/css?family=Lato:400,700,800,900|Manuale:400,700');
```

Comment:
Do we need to import 800 and 900 for Manuale as well if those font weights are available?

Reply:
Looks like [Manuale](https://fonts.google.com/specimen/Manuale) only goes to 700.